bin/convert2vulyk.py

```diff
@@ -146,58 +289,61 @@ def convert(text_files, ignore_annotations, ann_autodiscovery):
             else:
                 markup = ann.read_text()
 
-        vulyk_obj = convert_bsf_2_vulyk(text.read_text(), markup)
+        tokenized: list[list[str]] = read_and_tokenize(
+            text.read_text(), fmt, TokenizationType.NOOP if fmt == "json" else TokenizationType.WHITESPACE
+        )
+
+        vulyk_obj: dict = convert_bsf_2_vulyk(tokenized, markup, compensate_for_offsets=True)
         print(json.dumps(vulyk_obj, ensure_ascii=False, sort_keys=True))
 
 
-def convert_command(args):
+def convert_command(args: argparse.Namespace) -> None:
     return convert(
-        text_files=args.text_files,
+        input_files=args.input_files,
+        fmt=args.format,
         ignore_annotations=args.ignore_annotations,
         ann_autodiscovery=args.ann_autodiscovery,
     )
 
 
-def tag(text_files, ner_framework, ner_model):
+def tag(input_files: str, fmt: str, ner_framework: str, ner_model: str) -> None:
     if ner_framework == "stanza":
-        model = StanzaNER(ner_model)
+        model: AbstractNER = StanzaNER(ner_model)
     elif ner_framework == "spacy":
         model = SpacyNER(ner_model)
```

Comment:
to be consistent this line should be `model: AbstractNER`

Reply:
It is, no?

Reply:
What I meant is that in 311 you added type def but in 313 it is missing

Reply:
<img width="963" alt="image" src="https://user-images.githubusercontent.com/131186/152796480-fcbbe9ba-6ce0-40ef-a395-ad73577cf1eb.png">
mypy: error
no-redef - Name "model" already defined on line 311

Reply:
Oh yes, you're right, python scope is within function not within the nearest block.

Btw, I've played a bit with type checking in VS Code and 
<img width="851" alt="Screenshot 2022-02-09 at 11 42 14" src="https://user-images.githubusercontent.com/1867895/153170130-4b6a0d44-25f3-4e6e-85fa-6cc03289514a.png">
even when setting type check to basic - it shows 17 issues for me. With strict - it goes to 81 issues.

Reply:
For some reason Pylance still prefers typing.List over list even though the first is not recommended